README.md

```diff
@@ -58,13 +58,18 @@ This card can be configured using Lovelace UI editor.
 5. Choose `entity`.
 6. Now you should see the preview of the card!
 
-_Sorry, no support for `actions` and `stats` in visual config yet._
+_Sorry, no support for `start_action`, `actions` and `stats` in visual config yet._
 
 Typical example of using this card in YAML config would look like this:
 
 ```yaml
 type: 'custom:vacuum-card'
 entity: vacuum.vacuum_cleaner
+start_action:
```

Comment:
Please update cover `shortcuts` and `actions` in documentation

Reply:
I have updated documentation, let me know if it correct for you

Reply:
Hi @denysdovhan, can we resolve this?